nextroute/schema/output.go

```diff
@@ -2,13 +2,16 @@ package schema
 
 import (
 	"time"
+
+	"github.com/nextmv-io/sdk/nextroute"
```

Comment:
Linked to my comment about placing the check stuff in its own package, notice here that `schema` is importing `nextroute`. I consider `schema` a sub-package of `nextroute` so I find it strange that `schema` has to import its parent package. If anything, it should be the other way around, where necessary.